homework/index.js

```diff
@@ -28,14 +28,42 @@
     });
     return elem;
   }
+  function createTable(table, header, value) {
+    const tr = createAndAppend('tr', table);
+    createAndAppend('th', tr, { text: header, class: 'keys' });
+    createAndAppend('td', tr, { text: value, class: 'values' });
+    return tr;
+  }
+
+  function changDate(dateTime) {
```

Comment:
This is not the best possible name. You are not changing the date, you are just formatting an existing date for presentation purposes. So `formatDate` would be a better name.